Formula/sdl.rb

```diff
@@ -26,6 +27,34 @@ class Sdl < Formula
         sha256 "954875a277d9246bcc444b4e067e75c29b7d3f3d2ace5318a6aab7d7a502f740"
       end
     end
+
+    # Fix display issues on 10.14+, https://bugzilla.libsdl.org/show_bug.cgi?id=4788
```

Comment:
Can this work with just the raw diff of the patches here?

Reply:
I'm not 100% sure I know what you're asking here.  If you're asking whether it builds correctly with these changes, this formula does successfully build and resolves the issue on my Catalina machine.  If you're asking about collapsing the diffs to reduce the number of individual patches that need to be made, I don't see a way to do that on their Mercurial web UI.  I might be missing something with that, though.

Reply:
It's the latter, and if you go to the attached patch you can view a "raw unified diff" which would be usable as a patch AFAIK.

Reply:
Oh, the attachment on the Bugzilla tracker.  Gotcha.  Sure, I just added another commit with that one instead.